components/gh_events/GitHubEvent.tsx

```diff
@@ -15,45 +25,48 @@ export default function GitHubEvent({ event }: { event?: IGitHubEvent }) {
   switch (event.type) {
     case "MemberEvent":
       title = (
-        <>
+        <div className="flex gap-1 items-center">
+          <GoPersonAdd className="text-blue-500 text-lg" />
```

Comment:
Let's keep the colors of the icon the same or in line with the theme. We are not using this blue anywhere else IIRC.

I'd prefer to go with the same color as the title text.